src/utils/utils.js

```diff
@@ -123,6 +123,13 @@ export const leftDeepOnlyInlineInScopePath = createDOMPathGenerator(
     true,
     true,
 );
+export const leftDeepOnlyInlineInScopeNoEditable = createDOMPathGenerator(
+    DIRECTIONS.LEFT,
+    true,
+    true,
+    true,
+    node => node.getAttribute && node.getAttribute('contenteditable') === 'false',
```

Comment:
Why do we need this for "left deep" and not for the other paths ?

Reply:
I've not come across the need to check if the contenteditable is false for other cases.
I check for the right deep but it does not seems to matter to check the contenteditable there.

Reply:
That's weird isn't it? :woman_shrugging: